targets/TARGET_STM/TARGET_STM32L0/TARGET_MCU_STM32L071xx/device/TOOLCHAIN_GCC_ARM/startup_stm32l071xx.S

```diff
@@ -1,29 +1,42 @@
-/* mbed Microcontroller Library
- * SPDX-License-Identifier: BSD-3-Clause
- ******************************************************************************
- *
- * Copyright (c) 2020 STMicroelectronics. 
- * All rights reserved.
- *
- * This software component is licensed by ST under BSD 3-Clause license,
- * the "License"; You may not use this file except in compliance with the 
- * License. You may obtain a copy of the License at:
- *                        opensource.org/licenses/BSD-3-Clause
- *
- ******************************************************************************
- * @file      startup_stm32l071xx.s
- * @author    MCD Application Team
- * @brief     STM32L071xx Devices vector table for GCC toolchain.
- *            This module performs:
- *                - Set the initial SP
- *                - Set the initial PC == Reset_Handler,
- *                - Set the vector table entries with the exceptions ISR address
- *                - Branches to main in the C library (which eventually
- *                  calls main()).
- *            After Reset the Cortex-M0+ processor is in Thread mode,
- *            priority is Privileged, and the Stack is set to Main.
- ******************************************************************************
- */
+/**
+  ******************************************************************************
+  * @file      startup_stm32l073xx.s
```

Comment:
wrong file...

Reply:
when I use the file provided here https://github.com/STMicroelectronics/STM32CubeL0/blob/master/Drivers/CMSIS/Device/ST/STM32L0xx/Source/Templates/gcc/startup_stm32l071xx.s nothing works, I know that it is wrong, but I was able to successfully upload and make the MCU work with the startup file from the nucleol073rz provided in mbed-os target directory.

Reply:
@jeromecoutant can you tell me if this is now ok?

Reply:
It should work with the file from ST **AND** the bl_start call patch